lib/apis/v2/policy.go

```diff
@@ -162,18 +163,27 @@ type EntityRule struct {
 	// The effect is that the latter will accept packets from non-Calico sources whereas the
 	// former is limited to packets from Calico-controlled endpoints.
 	Selector string `json:"selector,omitempty" validate:"omitempty,selector"`
+
+	// NamespaceSelector is an optional field that contains a selector expression. Only traffic
+	// that originates from (or terminates at) endpoints within the selected namespaces will be
+	// matched.
```

Comment:
How is this defaulted for GNP and NP?

Reply:
For both it is defaulted to `""`, which indicates not to perform any namespace selection.

The implied behavior is different for the two resources:
- NP - implies select only within my own namespace
- GNP - implies select across all namespaces.

Reply:
Sorry, was meaning, would it make sense to add a note to the comment to explain that `""` means different things in different types of policy?

Reply:
ah, yeah. can do.